src/cmd/delegation_backend/main_bpu.go

```diff
@@ -65,3 +80,10 @@ func main() {
 	}()
 	log.Fatal(http.ListenAndServe(DELEGATION_BACKEND_LISTEN_TO, nil))
 }
+
+func GetAWSBucketName(config AppConfig) string {
```

Comment:
I do not agree that this getter function should be in main. `app_config.go` would be a better place for it. Any additional rationale behind it I don't know of?

Reply:
Ok, moved back.